tests/fixtures/test_robot/test_tags_with_kwargs/conftest.py

```diff
@@ -0,0 +1,10 @@
+from pytest_robotframework._internal.pytest_robot_items import RobotItem
+
+
+def pytest_runtest_setup(item: RobotItem):
+    assert item.name == "Foo"
+    marker_names = ["m1", "m2"]
+    marker_kwargs = [{"foo": "1", "baz": "false"}, {"bar": "foo", "qux": "7"}]
+    for i, marker in enumerate(item.iter_markers()):
+        assert marker.name == marker_names[i]
+        assert marker.kwargs == marker_kwargs[i]  # type: ignore[no-any-expr]
```

Comment:
this won't fail if there are no markers. maybe do this instead
```suggestion
    for marker, marker_name, marker_kwarg in zip(item.iter_markers(), marker_names, marker_kwargs, strict=True):
        assert marker.name == marker_name
        assert marker.kwargs == marker_kwarg  # type: ignore[no-any-expr]
```
specifically the `strict=True` will make sure all the lists being iterated over are the same length